From fcb4122e486824a5ac582d15076efa79a5f23f3a Mon Sep 17 00:00:00 2001
Message-ID: <fcb4122e486824a5ac582d15076efa79a5f23f3a.1740173176.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1740173176.git.sharlatanus@gmail.com>
References: <cover.1740173176.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Mon, 17 Feb 2025 18:52:53 +0000
Subject: [PATCH 26/51] gnu: python-pyvo: Update to 1.6.1.

* gnu/packages/astronomy.scm (python-pyvo): Update to 1.6.1.
[propagated-inputs]: Add python-defusedxml.

Change-Id: I9aedd6c2f6f9459739ae65505035bee61531afa7
---
 gnu/packages/astronomy.scm | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 1d252cc6f45..04e7bf7925b 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -4783,13 +4783,13 @@ (define-public python-pyavm
 (define-public python-pyvo
   (package
     (name "python-pyvo")
-    (version "1.6")
+    (version "1.6.1")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "pyvo" version))
        (sha256
-        (base32 "1ahcazgxhmxscjgz58mxw75k9g0xm7iffnc4lqfbm3h0wqiz8vnv"))))
+        (base32 "0hfdsb6yqda6g987b96jqaw4kqj4qh9qz3am6inx0nl2m5y735pq"))))
     (build-system pyproject-build-system)
     (arguments
      (list
@@ -4802,7 +4802,8 @@ (define-public python-pyvo
            python-wheel))
     (propagated-inputs
      (list python-astropy
-           python-pillow
+           python-defusedxml ; extra requirements
+           python-pillow     ; extra requirements
            python-requests))
     (home-page "https://github.com/astropy/pyvo")
     (synopsis "Access Virtual Observatory data and services")
-- 
2.47.1

